lib/ui/views/about/about_view.dart

```diff
@@ -50,7 +50,7 @@ class _AboutViewState extends State<AboutView> {
   }
 
   Widget _buildContributorsList() {
-    switch (_model.stateFor(_model.FETCH_CONTRIBUTORS)) {
+    switch (_model.stateFor(_model.fetchCONTRIBUTORS)) {
```

Comment:
https://github.com/CircuitVerse/mobile-app/pull/177#discussion_r762757887

Reply:
Check if you can avoid renaming the variables while supporting the rule.

Reply:
I'm not quite sure what you meant here😅. Do I have to revert the changes? Or add ignore statements at a few places?

Reply:
We want to keep using UPPER_CASE for the current variables we have and while supporting `non_constant_identifier_names`.

You need make UPPER_CASE variables as `const` and keep them in UPPER_CASE. In this way the rule will not be causing issue since we made it a constant.